openslides/core/views.py

```diff
@@ -115,14 +114,9 @@ def get_context_data(self, **kwargs):
         Adds version strings to the context.
         """
         context = super().get_context_data(**kwargs)
-        # OpenSlides version. During development the git commit id is added.
-        if RELEASE:
-            description = ''
-        else:
-            description = 'Commit %s' % get_git_commit_id()
         context['modules'] = [{'verbose_name': 'OpenSlides',
-                               'description': description,
-                               'version': get_openslides_version()}]
+                               'description': '',
```

Comment:
I think this can be removed. But we have to rewrite this view with angular in any way. For that reason we need an ajax-view to get the version. If you like, you could create it with this pull request.